tests/by-util/test_tail.rs

```diff
@@ -148,6 +150,7 @@ fn test_stdin_redirect_offset2() {
         .succeeded();
 }
 
+// TODO: Use a debug_assert() instead
```

Comment:
Instead of what? Also `debug_assert` is not very meaningful in tests. Tests are not included in regular binaries and should probably always fail early if something yas gone wrong, even when run with `--release`

Reply:
Sorry, seems my TODOs are only meant for me. This is a quick one about adding a debug_assert() in tail.rs() after the immediate exit because of `-[nc]0` instead of trying to provoke an error like passing a missing file to tail. A debug_assert() is more precise in my eyes.